frontend/src/pages/askGpt/AskGpt.jsx

```diff
@@ -2,6 +2,7 @@ import React, { useState } from 'react'
 import './askGpt.css'
 
 function AskGpt() {
+    const url="https://legalassist.onrender.com"
```

Comment:
Nice change, this could also be an environment variable for the front-end app. So that you can easily change its value in a `.env` file instead of the code